packages/@aws-cdk/aws-ecs-patterns/test/fargate/test.load-balanced-fargate-service.ts

```diff
@@ -228,6 +229,24 @@ export = {
     test.done();
   },
 
+  'setting platform version'(test: Test) {
```

Comment:
We have a test called `'test Fargate loadbalanced construct with all settings'` in which we specify every supported property. Updating that test should be sufficient.

Reply:
That test is in a different file for a different construct (`test.load-balanced-fargate-service-v2.ts`). I did use the test for every property in that one, but I didn't see anything like that in this file.

Reply:
If you can add the test in that file, that would be great! (also if you can add a test for all 4 load balanced service constructs that would be awesome!)